src/r_vector/vectors.rs

```diff
@@ -279,6 +279,21 @@ impl From<Vec<String>> for Vector {
     }
 }
 
+impl Into<Vec<String>> for Vector {
+    fn into(self) -> Vec<String> {
+        match self {
```

Comment:
We can support more vector types by automatically doing the coercion before extracting string representations.

```suggestion
        match self.as_character() {
```